doc/grpc_release_schedule.md

```diff
@@ -0,0 +1,22 @@
+# gRPC Release Schedule
+
+Below is the release schedule for gRPC [Java](https://github.com/grpc/grpc-java/releases), [Go](https://github.com/grpc/grpc-go/releases) and [Core](https://github.com/grpc/grpc/releases) and its  dependent languages C++, C#, Objective-C, PHP, Python and Ruby.
+
+Releases are scheduled every six weeks on Tuesdays on a best effort basis. In some unavoidable situations a release may be delayed or a language may skip a release altogether and do the next release to catch up with other languages. See the past releases in the links above.
+
+Releases are done off of release branches. For Core and Java repos, the branch is cut two weeks before the scheduled release date. An RC (release candidate) is published for Core and its dependent languages just after the branch cut.  We do our best to keep head of master branch stable at all times regardless of release schedule. Daily build packages from master branch for C#, PHP, Python, Ruby and Protoc plugins are published on [grpc.io](https://packages.grpc.io/)
```

Comment:
Releases are cut from release branches.

Reply:
Do we want to state that RCs are promoted to releases? 
It is useful to add that if you depend on gRPC in production, you should test the RCs.